### PR TITLE
Mobile subpage optimizations

### PR DESCRIPTION
**Pull request by eadams17**
## WHAT
optimize teacher/admin dashboard subpages to be responsive

## WHY
this is the last step to make Quill usable on tablet/mobile!

## HOW
a bunch of CSS changes

### Screenshots
(If applicable. Also, please censor any sensitive data)
Screen Shot 2023-05-19 at 3 35 38 PM
Screen Shot 2023-05-19 at 3 37 02 PM
Screen Shot 2023-05-19 at 3 37 28 PM
Screen Shot 2023-05-19 at 3 37 51 PM
Screen Shot 2023-05-19 at 3 38 40 PM
Screen Shot 2023-05-19 at 3 39 16 PM
Screen Shot 2023-05-19 at 3 39 44 PM
Screen Shot 2023-05-19 at 3 40 09 PM
Screen Shot 2023-05-19 at 3 40 24 PM
Screen Shot 2023-05-19 at 3 40 40 PM

### Notion Card Links
(Please provide links to any relevant Notion card(s) relevant to this PR.)
https://www.notion.so/quill/Optimize-subpages-for-smaller-screen-sizes-0bab61d758ee4dadb4edf40620ac78af

PR Checklist | Your Answer
------------ | -------------
Have you added and/or updated tests? |  yes
Have you deployed to Staging? | yes
Self-Review: Have you done an initial self-review of the code below on Github? | yes
Spec Review: Have you reviewed the spec and ensured this meets requirements and/or matches design mockups? | yes

### CODE REVIEW

**Review by emilia-friedberg — APPROVED** [review]
Code looks good to me! I think there are some additional styling tweaks I would make, just based on the screenshots, but you can take or leave:

- On the subscriptions page, I'd break the school image onto its own line after the section header (or hide it), and make sure all the text is left-justified.
- On the active classes page, I'd stack each of the classroom attributes vertically so we don't end up with things like "co-teacher" getting wrapped in the middle of the word.
- On the Data Export page, I'd stack the dropdowns vertically instead of having them next to each other so the text doesn't get cut off. I think it also might be a good idea to try to give that table a minimum width so the column headers don't get so cut off.

File: services/QuillLMS/client/app/bundles/PremiumHub/components/concept_reports_table.jsx [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { ReactTable, } from '../../Shared';
 import EmptyStateForReport from '../../Teacher/components/progress_reports/empty_state_for_report';
 
 const ConceptReportsTable = ({ data, }) => {
+  console.log("🚀 ~ file: concept_reports_table.jsx:8 ~ ConceptReportsTable ~ data:", data)
```
Comment (anathomical) [inline services/QuillLMS/client/app/bundles/PremiumHub/components/concept_reports_table.jsx]: Do we want this `console.log` call to stay in?

**Review by anathomical — APPROVED** [review]
Looks fine to me.  One question about a `console.log` call

File: services/QuillLMS/client/app/bundles/PremiumHub/components/concept_reports_table.jsx (hunk 1 above)
Comment (eadams17) [reply services/QuillLMS/client/app/bundles/PremiumHub/components/concept_reports_table.jsx]: @anathomical no, I missed that-- thanks for catching!

**Pull request merged**